File: app/src/main/java/net/emojiparty/android/jishotomo/ui/presentation/StringForJlptLevel.java
package net.emojiparty.android.jishotomo.ui.presentation;

import android.content.Context;

public class StringForJlptLevel {
  public static int getId(Integer level, Context context) {
    return context.getResources()
        .getIdentifier("jlpt_n" + level, "string", context.getPackageName());
  }
}

File: app/src/main/java/net/emojiparty/android/jishotomo/ui/presentation/StringForJlptLevel.kt
package net.emojiparty.android.jishotomo.ui.presentation

import android.content.Context

object StringForJlptLevel {
  fun getId(
    level: Int,
    context: Context
  ): Int {
    return context.resources.getIdentifier("jlpt_n$level", "string", context.packageName)
  }
}
